src/skin/js/pb.js

```diff
@@ -0,0 +1,15 @@
+(function($) {
+
+$(window).on("load", function () {
+
+  function setSeenComic() {
+    var badger = chrome.extension.getBackgroundPage().badger;
+    var settings = badger.storage.getBadgerStorageObject("settings_map");
+    settings.setItem("seenComic", true);
+  }
+
+  $(".scroll-it").smoothScroll({afterScroll: function() {  setSeenComic; }});
```

Comment:
I think we need to call the function here (or remove the surrounding anonymous function).

Reply:
The anonymous function is part of the syntax for smoothScroll's afterscroll.

Reply:
How soon does `afterScroll` trigger? If `afterScroll` is triggered by any amount of scrolling, we would need to do more to avoid setting `seenComic` too early. Should we wait until the user got to the main explanatory portion of the walkthrough?

Reply:
The problem is that when `afterScroll` triggers, it calls the anonymous function which references but does not call `setSeenComic`. So nothing actually happens. You would fix it by either calling `setSeenComic` or removing the anonymous function, which will then have `afterScroll` call `setSeenComic` directly. Either of the following lines should do it:
```javascript
$(".scroll-it").smoothScroll({
  afterScroll: function () {
    setSeenComic();
  }
});
```
```javascript
$(".scroll-it").smoothScroll({
  afterScroll: setSeenComic
});
```

Reply:
Oh right, it's missing the () after the function call in my version.